openapi/kas-fleet-manager.yaml

```diff
@@ -1065,23 +1065,24 @@ components:
                 allOf:
                   - $ref: "#/components/schemas/KafkaRequest"
     VersionMetadata:
-      allOf:
-        - $ref: "#/components/schemas/ObjectReference"
-        - type: object
-          example:
-            kind: "APIVersion"
-            id: "v1"
-            href: "/api/kafkas_mgmt/v1"
-            collections:
-              - id: "kafkas"
-                href: "/api/kafkas_mgmt/v1/kafkas"
-                kind: "KafkaList"
-          properties:
-            collections:
-              type: array
-              items:
-                allOf:
-                  - $ref: "#/components/schemas/ObjectReference"
+      required:
+        - kind
+        - id
+        - href
+      type: object
+      properties:
```

Comment:
There is no need to duplicate the kind, id and href properties. The `VersionMetadata` object can be defined as a Composition of the `ObjectReference` object and a custom object inline that has the server_version and the collections attributes

Reply:
Thanks @miguelsorianod ! The changes were implemented in the last commit.